src/ldp/graph/ops.py

```diff
@@ -246,7 +246,7 @@ def get_upstream_results(self, op: str | Op) -> Iterator[OpResult]:
 
     def traverse(
         self,
-        topological_order: bool = True,
+        topological_order: bool = False,
```

Comment:
I'm cool with changing the default, but can you pass `topological_order=True` here: https://github.com/Future-House/ldp/blob/3d874c59088f70efbe323878bd52b748438bcb50/src/ldp/graph/ops.py#L119 ? That's the only place we currently require topological sorting.